core/controllers/subscriptions.py

```diff
@@ -21,26 +21,51 @@
 from core.controllers import base
 from core.domain import subscription_services
 from core.domain import user_services
+import feconf
 
 
 class SubscribeHandler(base.BaseHandler):
     """Handles operations relating to new subscriptions."""
 
+    POST_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'creator_username': {
+                'schema': {
+                    'type': 'basestring'
+                }
```

Comment:
There is a validation function for username, please add it here. You can look for it in other handlers that use username.

Reply:
I have added validation function